MobiFlight/ExecutionManager.cs

```diff
@@ -864,7 +864,7 @@ private string ExecuteDisplay(string value, OutputConfigItem cfg)
                         break;
 
                     case OutputConfig.CustomDevice.Type:
-                        mobiFlightCache.Set(serial, cfg.CustomDevice, value, GetRefs(cfg.ConfigRefs));

```

Comment:
Despite being passed in, `cfg.ConfigRefs` is never used by the method.

Reply:
I don't remember if that was some idea which we had at some point.